compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -659,7 +659,7 @@ impl<'a> Parser<'a> {
                                 Ok(_) => {
                                     e.span_suggestion_verbose(
                                         binop.span.shrink_to_lo(),
-                                        "use `::<...>` instead of `<...>` to specify type arguments",
+                                        "use `::<...>` instead of `<...>` to specify type or const arguments",
```

Comment:
```suggestion
                                        TURBOFISH,
```
or maybe even `TURBOFISH_SUGGESTION_STR` so it's a bit more obvious what it means